spec/rubocop/cop/chromebrew/ordered_compatibility_spec.rb

```diff
@@ -0,0 +1,41 @@
+RSpec.describe RuboCop::Cop::Chromebrew::OrderedCompatibility, :config do
+  it 'registers an offense when compatibiltiy is not in alphabetical order with three elements' do
+    expect_offense(<<~'RUBY')
+      compatibility 'x86_64 aarch64 armv7l'
+                    ^^^^^^^^^^^^^^^^^^^^^^^ Compatibility values should be in alphabetic order.
```

Comment:
There are many other ways the order can be incorrect.  Maybe it would make sense to just return an error if compatibility doesn't match any of the "acceptable" patterns?  Keep in mind, the compatibility line will still function correctly if double quotes instead of single quotes or commas instead of spaces are used.

Reply:
This is just the spec-- I didn't list every possible example, but I have also locally tested it on every single package file and it's worked fine.

Reply:
My point is `compatibility 'aarch64 x86_64 armv7l'`   and `compatibility 'armv7l aarch64 x86_64'` should all fail.  This testing should be based on the pr submitted, not the overall status of current packages.  If you compare this line to all possible valid compatibility values, the testing will be more thorough.

Reply:
> My point is compatibility 'aarch64 x86_64 armv7l' and compatibility 'armv7l aarch64 x86_64' should all fail.

They do fail, I just don't test them as part of the spec. If I listed every possible compatibility value, the spec would be impossible to read and take forever to run.

Reply:
I guess I'm not familiar with how a spec works.  We'll find out soon enough.